corehq/apps/hqwebapp/static/cloudcare/css/webforms.css

```diff
@@ -74,7 +74,7 @@ input:invalid {
 }
 
 .map {
-  width: 400px;
+  width: 230px;
```

Comment:
This definitely looks better in app preview. Have you looked at it in the "tablet" version of app preview (the bottom-right icon on the phone image, next to back & refresh) and in web apps? I'm wondering if it would be worth doing size-specific styling here. @biyeun might also have an opinion.

Reply:
Ahh I did not know this was used in other places. I think I got a demo of the web app once, but never figured out access to that. Can you tell me how I'd test on the "tablet" version and web apps?

Reply:
For tablet view, it's the tablet-y icon in the very bottom black area:
![Screen Shot 2020-07-07 at 12 13 19 PM](https://user-images.githubusercontent.com/1486591/86810895-5cc95500-c04b-11ea-9a31-46951651f3b3.png)

Then there's a link to "Web Apps" in the top nav. Feel free to offline with me if you don't see it, you might need to tweak your local configuration to make it visible.

Reply:
Awesome, thanks! Would changing it to `100%` work?

<img width="320" alt="Screen Shot 2020-07-07 at 1 49 31 PM" src="https://user-images.githubusercontent.com/615126/86821934-f9deba80-c058-11ea-91f9-41b62678f91b.png">

<img width="542" alt="Screen Shot 2020-07-07 at 1 49 42 PM" src="https://user-images.githubusercontent.com/615126/86821962-0105c880-c059-11ea-9917-f80abf9d50fe.png">


<img width="1082" alt="Screen Shot 2020-07-07 at 1 50 18 PM" src="https://user-images.githubusercontent.com/615126/86821976-0400b900-c059-11ea-9d48-5ccf3a8b2a58.png">

Reply:
That looks great, thank you for the screenshots.